lib/facility_factory.rb

```diff
@@ -0,0 +1,47 @@
+class FacilityFactory
+
+    def initialize
+    end
+
+    def construct_co_facility_attr_helper(office_data)
```

Comment:
similar to my comment above, we could just remove `_helper` from these method names in my opinion